composer.json

```diff
@@ -122,9 +122,10 @@
         "symfony/yaml": "self.version"
     },
     "require-dev": {
-        "amphp/amp": "^2.5",
-        "amphp/http-client": "^4.2.1",
-        "amphp/http-tunnel": "^1.0",
+        "amphp/amp": "^2.5|^3.x-dev",
+        "amphp/http-client": "^4.2.1|^5.x-dev",
+        "amphp/http-tunnel": "^1.0|^2.x-dev",
+        "amphp/parallel": "^1.x-dev|^2.x-dev",
```

Comment:
I suggest not changing this file until HttpClient is compatible with the latest version of AMPHP. This will simplify the CI.